buildbot/plot_profdata.py

```
@@ -38,12 +38,12 @@ def load_dic(fname):
 
 for dic in dic_lst:
     for k in dic.keys():
-        if not (k in dic_labels.keys()):
+        if not (k in dic_labels):
             dic_labels[k] = counter
             counter += 1
 
 # label list for the plot
-labels = [k for k in dic_labels.keys()]
+labels = [k for k in dic_labels]
```

Ruff (0.16.0)

[warning] Unnecessary list comprehension (rewrite using `list()`)

Rewrite using `list()`

(C416)